### DISCUSSION TITLE
Extra%20Chapters/Compute%20Shaders/

### DISCUSSION THREAD

**Discussion by giscus** [wmarti/MetalTutorial#5] [Announcements]
# Extra%20Chapters/Compute%20Shaders/

https://metaltutorial.com/Extra%20Chapters/Compute%20Shaders/

**Comment by codesavory** [comment]
I would love to see the tutorial and write-up showcasing the compute shader. Great tutorial so far! 

At least share the code and I can try to understand how it is done. Thank you!
